[llvm] Build failure

Package: llvm:x64-linux -> 15.0.7#4

**Host Environment**

- Host: x64-linux
- Compiler: GNU 11.4.0
-    vcpkg-tool version: 2999-12-31-unknownhash
    vcpkg-scripts version: aab126361 2023-08-27 (20 minutes ago)

**To Reproduce**

`vcpkg test-features --all --failure-logs=/mnt/vss/_work/1/a/failure-logs --ci-feature-baseline=/mnt/vss/_work/1/s/scripts/azure-pipelines/../ci.feature.baseline.txt`

**Failure logs**

```
-- Downloading https://github.com/llvm/llvm-project/archive/llvmorg-15.0.7.tar.gz -> llvm-llvm-project-llvmorg-15.0.7.tar.gz...
-- Extracting source /mnt/vcpkg-ci/downloads/llvm-llvm-project-llvmorg-15.0.7.tar.gz
-- Applying patch 0001-Fix-install-paths.patch
-- Applying patch 0002-Fix-DR-1734.patch
-- Applying patch 0003-Fix-tools-path.patch
-- Applying patch 0004-Fix-compiler-rt-install-path.patch
-- Applying patch 0005-Fix-tools-install-path.patch
-- Applying patch 0007-Fix-install-bolt.patch
-- Applying patch 0008-llvm_assert.patch
-- Applying patch 0009-disable-libomp-aliases.patch
-- Using source at /mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean
-- Getting CMake variables for x64-linux
-- Configuring x64-linux
CMake Warning at /mnt/vcpkg-ci/installed/x64-linux/share/vcpkg-cmake/vcpkg_cmake_configure.cmake:318 (message):
  The following variables are not used in CMakeLists.txt:

      BOLT_TOOLS_INSTALL_DIR
      CLANG_TOOLS_INSTALL_DIR
      LIBOMP_INSTALL_ALIASES
      LLD_TOOLS_INSTALL_DIR

  Please recheck them and remove the unnecessary options from the
  `vcpkg_cmake_configure` call.

  If these options should still be passed for whatever reason, please use the
  `MAYBE_UNUSED_VARIABLES` argument.
Call Stack (most recent call first):
  ports/llvm/portfile.cmake:237 (vcpkg_cmake_configure)
  scripts/ports.cmake:147 (include)


-- Building x64-linux-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: /mnt/vcpkg-ci/downloads/tools/cmake-3.27.1-linux/cmake-3.27.1-linux-x86_64/bin/cmake --build . --config Debug --target install -- -v -j33
    Working Directory: /mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg
    See logs for more information:
      /mnt/vcpkg-ci/buildtrees/llvm/install-x64-linux-dbg-out.log

Call Stack (most recent call first):
  /mnt/vcpkg-ci/installed/x64-linux/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  /mnt/vcpkg-ci/installed/x64-linux/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  ports/llvm/portfile.cmake:270 (vcpkg_cmake_install)
  scripts/ports.cmake:147 (include)



```
<details><summary>/mnt/vcpkg-ci/buildtrees/llvm/install-x64-linux-dbg-out.log</summary>

```
Change Dir: '/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg'

Run Build Command(s): /mnt/vcpkg-ci/downloads/tools/ninja/1.10.2-linux/ninja -v -v -j33 install
[1/1678] cd /mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg && /mnt/vcpkg-ci/downloads/tools/cmake-3.27.1-linux/cmake-3.27.1-linux-x86_64/bin/cmake -E make_directory /mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/NATIVE
[2/1678] /usr/bin/c++ -DGTEST_HAS_RTTI=0 -D_DEBUG -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/lib/Support -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/Support -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/include -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/include -fPIC -fPIC -fno-semantic-interposition -fvisibility-inlines-hidden -Werror=date-time -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wno-missing-field-initializers -pedantic -Wno-long-long -Wimplicit-fallthrough -Wno-maybe-uninitialized -Wno-class-memaccess -Wno-redundant-move -Wno-pessimizing-move -Wno-noexcept-type -Wdelete-non-virtual-dtor -Wsuggest-override -Wno-comment -Wmisleading-indentation -fdiagnostics-color -g -std=c++14  -fno-exceptions -fno-rtti -MD -MT lib/Support/CMakeFiles/LLVMSupport.dir/ABIBreak.cpp.o -MF lib/Support/CMakeFiles/LLVMSupport.dir/ABIBreak.cpp.o.d -o lib/Support/CMakeFiles/LLVMSupport.dir/ABIBreak.cpp.o -c /mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/Support/ABIBreak.cpp
[3/1678] /usr/bin/c++ -DGTEST_HAS_RTTI=0 -D_DEBUG -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/lib/Support -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/Support -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/include -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/include -fPIC -fPIC -fno-semantic-interposition -fvisibility-inlines-hidden -Werror=date-time -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wno-missing-field-initializers -pedantic -Wno-long-long -Wimplicit-fallthrough -Wno-maybe-uninitialized -Wno-class-memaccess -Wno-redundant-move -Wno-pessimizing-move -Wno-noexcept-type -Wdelete-non-virtual-dtor -Wsuggest-override -Wno-comment -Wmisleading-indentation -fdiagnostics-color -g -std=c++14  -fno-exceptions -fno-rtti -MD -MT lib/Support/CMakeFiles/LLVMSupport.dir/COM.cpp.o -MF lib/Support/CMakeFiles/LLVMSupport.dir/COM.cpp.o.d -o lib/Support/CMakeFiles/LLVMSupport.dir/COM.cpp.o -c /mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/Support/COM.cpp
[4/1678] /usr/bin/c++ -DGTEST_HAS_RTTI=0 -D_DEBUG -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/lib/Support -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/Support -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/include -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/include -fPIC -fPIC -fno-semantic-interposition -fvisibility-inlines-hidden -Werror=date-time -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wno-missing-field-initializers -pedantic -Wno-long-long -Wimplicit-fallthrough -Wno-maybe-uninitialized -Wno-class-memaccess -Wno-redundant-move -Wno-pessimizing-move -Wno-noexcept-type -Wdelete-non-virtual-dtor -Wsuggest-override -Wno-comment -Wmisleading-indentation -fdiagnostics-color -g -std=c++14  -fno-exceptions -fno-rtti -MD -MT lib/Support/CMakeFiles/LLVMSupport.dir/AutoConvert.cpp.o -MF lib/Support/CMakeFiles/LLVMSupport.dir/AutoConvert.cpp.o.d -o lib/Support/CMakeFiles/LLVMSupport.dir/AutoConvert.cpp.o -c /mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/Support/AutoConvert.cpp
...
Skipped 1964 lines
...

/usr/bin/c++ -DGTEST_HAS_RTTI=0 -D_DEBUG -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/lib/Debuginfod -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/Debuginfod -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/include -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/include -fPIC -fPIC -fno-semantic-interposition -fvisibility-inlines-hidden -Werror=date-time -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wno-missing-field-initializers -pedantic -Wno-long-long -Wimplicit-fallthrough -Wno-maybe-uninitialized -Wno-class-memaccess -Wno-redundant-move -Wno-pessimizing-move -Wno-noexcept-type -Wdelete-non-virtual-dtor -Wsuggest-override -Wno-comment -Wmisleading-indentation -fdiagnostics-color -g -std=c++14  -fno-exceptions -fno-rtti -MD -MT lib/Debuginfod/CMakeFiles/LLVMDebuginfod.dir/Debuginfod.cpp.o -MF lib/Debuginfod/CMakeFiles/LLVMDebuginfod.dir/Debuginfod.cpp.o.d -o lib/Debuginfod/CMakeFiles/LLVMDebuginfod.dir/Debuginfod.cpp.o -c /mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/Debuginfod/Debuginfod.cpp
/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/Debuginfod/Debuginfod.cpp: In member function ‘llvm::Error llvm::DebuginfodCollection::updateForever(std::chrono::milliseconds)’:
/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/Debuginfod/Debuginfod.cpp:298:23: error: ‘sleep_for’ is not a member of ‘std::this_thread’
  298 |     std::this_thread::sleep_for(Interval);
      |                       ^~~~~~~~~
[1360/1678] /usr/bin/c++ -DGTEST_HAS_RTTI=0 -D_DEBUG -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/lib/DebugInfo/DWARF -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/DebugInfo/DWARF -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/include -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/include -fPIC -fPIC -fno-semantic-interposition -fvisibility-inlines-hidden -Werror=date-time -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wno-missing-field-initializers -pedantic -Wno-long-long -Wimplicit-fallthrough -Wno-maybe-uninitialized -Wno-class-memaccess -Wno-redundant-move -Wno-pessimizing-move -Wno-noexcept-type -Wdelete-non-virtual-dtor -Wsuggest-override -Wno-comment -Wmisleading-indentation -fdiagnostics-color -g -std=c++14  -fno-exceptions -fno-rtti -MD -MT lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFGdbIndex.cpp.o -MF lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFGdbIndex.cpp.o.d -o lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFGdbIndex.cpp.o -c /mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/DebugInfo/DWARF/DWARFGdbIndex.cpp
[1361/1678] /usr/bin/c++ -DGTEST_HAS_RTTI=0 -D_DEBUG -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/lib/DebugInfo/DWARF -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/DebugInfo/DWARF -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/include -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/include -fPIC -fPIC -fno-semantic-interposition -fvisibility-inlines-hidden -Werror=date-time -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wno-missing-field-initializers -pedantic -Wno-long-long -Wimplicit-fallthrough -Wno-maybe-uninitialized -Wno-class-memaccess -Wno-redundant-move -Wno-pessimizing-move -Wno-noexcept-type -Wdelete-non-virtual-dtor -Wsuggest-override -Wno-comment -Wmisleading-indentation -fdiagnostics-color -g -std=c++14  -fno-exceptions -fno-rtti -MD -MT lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFDebugMacro.cpp.o -MF lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFDebugMacro.cpp.o.d -o lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFDebugMacro.cpp.o -c /mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/DebugInfo/DWARF/DWARFDebugMacro.cpp
[1362/1678] /usr/bin/c++ -DGTEST_HAS_RTTI=0 -D_DEBUG -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/lib/DebugInfo/DWARF -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/DebugInfo/DWARF -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/include -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/include -fPIC -fPIC -fno-semantic-interposition -fvisibility-inlines-hidden -Werror=date-time -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wno-missing-field-initializers -pedantic -Wno-long-long -Wimplicit-fallthrough -Wno-maybe-uninitialized -Wno-class-memaccess -Wno-redundant-move -Wno-pessimizing-move -Wno-noexcept-type -Wdelete-non-virtual-dtor -Wsuggest-override -Wno-comment -Wmisleading-indentation -fdiagnostics-color -g -std=c++14  -fno-exceptions -fno-rtti -MD -MT lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFTypePrinter.cpp.o -MF lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFTypePrinter.cpp.o.d -o lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFTypePrinter.cpp.o -c /mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/DebugInfo/DWARF/DWARFTypePrinter.cpp
[1363/1678] /usr/bin/c++ -DGTEST_HAS_RTTI=0 -D_DEBUG -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/lib/DebugInfo/GSYM -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/DebugInfo/GSYM -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/include -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/include -fPIC -fPIC -fno-semantic-interposition -fvisibility-inlines-hidden -Werror=date-time -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wno-missing-field-initializers -pedantic -Wno-long-long -Wimplicit-fallthrough -Wno-maybe-uninitialized -Wno-class-memaccess -Wno-redundant-move -Wno-pessimizing-move -Wno-noexcept-type -Wdelete-non-virtual-dtor -Wsuggest-override -Wno-comment -Wmisleading-indentation -fdiagnostics-color -g -std=c++14  -fno-exceptions -fno-rtti -MD -MT lib/DebugInfo/GSYM/CMakeFiles/LLVMDebugInfoGSYM.dir/FileWriter.cpp.o -MF lib/DebugInfo/GSYM/CMakeFiles/LLVMDebugInfoGSYM.dir/FileWriter.cpp.o.d -o lib/DebugInfo/GSYM/CMakeFiles/LLVMDebugInfoGSYM.dir/FileWriter.cpp.o -c /mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/DebugInfo/GSYM/FileWriter.cpp
[1364/1678] /usr/bin/c++ -DGTEST_HAS_RTTI=0 -D_DEBUG -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/lib/DebugInfo/GSYM -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/DebugInfo/GSYM -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/include -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/include -fPIC -fPIC -fno-semantic-interposition -fvisibility-inlines-hidden -Werror=date-time -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wno-missing-field-initializers -pedantic -Wno-long-long -Wimplicit-fallthrough -Wno-maybe-uninitialized -Wno-class-memaccess -Wno-redundant-move -Wno-pessimizing-move -Wno-noexcept-type -Wdelete-non-virtual-dtor -Wsuggest-override -Wno-comment -Wmisleading-indentation -fdiagnostics-color -g -std=c++14  -fno-exceptions -fno-rtti -MD -MT lib/DebugInfo/GSYM/CMakeFiles/LLVMDebugInfoGSYM.dir/Header.cpp.o -MF lib/DebugInfo/GSYM/CMakeFiles/LLVMDebugInfoGSYM.dir/Header.cpp.o.d -o lib/DebugInfo/GSYM/CMakeFiles/LLVMDebugInfoGSYM.dir/Header.cpp.o -c /mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/DebugInfo/GSYM/Header.cpp
[1365/1678] /usr/bin/c++ -DGTEST_HAS_RTTI=0 -D_DEBUG -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/lib/DebugInfo/DWARF -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/DebugInfo/DWARF -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/include -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/include -fPIC -fPIC -fno-semantic-interposition -fvisibility-inlines-hidden -Werror=date-time -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wno-missing-field-initializers -pedantic -Wno-long-long -Wimplicit-fallthrough -Wno-maybe-uninitialized -Wno-class-memaccess -Wno-redundant-move -Wno-pessimizing-move -Wno-noexcept-type -Wdelete-non-virtual-dtor -Wsuggest-override -Wno-comment -Wmisleading-indentation -fdiagnostics-color -g -std=c++14  -fno-exceptions -fno-rtti -MD -MT lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFUnitIndex.cpp.o -MF lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFUnitIndex.cpp.o.d -o lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFUnitIndex.cpp.o -c /mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/DebugInfo/DWARF/DWARFUnitIndex.cpp
[1366/1678] /usr/bin/c++ -DGTEST_HAS_RTTI=0 -D_DEBUG -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/lib/DebugInfo/DWARF -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/DebugInfo/DWARF -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/include -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/include -fPIC -fPIC -fno-semantic-interposition -fvisibility-inlines-hidden -Werror=date-time -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wno-missing-field-initializers -pedantic -Wno-long-long -Wimplicit-fallthrough -Wno-maybe-uninitialized -Wno-class-memaccess -Wno-redundant-move -Wno-pessimizing-move -Wno-noexcept-type -Wdelete-non-virtual-dtor -Wsuggest-override -Wno-comment -Wmisleading-indentation -fdiagnostics-color -g -std=c++14  -fno-exceptions -fno-rtti -MD -MT lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFDebugLoc.cpp.o -MF lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFDebugLoc.cpp.o.d -o lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFDebugLoc.cpp.o -c /mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/DebugInfo/DWARF/DWARFDebugLoc.cpp
[1367/1678] /usr/bin/c++ -DGTEST_HAS_RTTI=0 -D_DEBUG -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/lib/DebugInfo/DWARF -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/DebugInfo/DWARF -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/include -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/include -fPIC -fPIC -fno-semantic-interposition -fvisibility-inlines-hidden -Werror=date-time -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wno-missing-field-initializers -pedantic -Wno-long-long -Wimplicit-fallthrough -Wno-maybe-uninitialized -Wno-class-memaccess -Wno-redundant-move -Wno-pessimizing-move -Wno-noexcept-type -Wdelete-non-virtual-dtor -Wsuggest-override -Wno-comment -Wmisleading-indentation -fdiagnostics-color -g -std=c++14  -fno-exceptions -fno-rtti -MD -MT lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFDebugRangeList.cpp.o -MF lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFDebugRangeList.cpp.o.d -o lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFDebugRangeList.cpp.o -c /mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/DebugInfo/DWARF/DWARFDebugRangeList.cpp
[1368/1678] /usr/bin/c++ -DGTEST_HAS_RTTI=0 -D_DEBUG -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/lib/DebugInfo/DWARF -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/DebugInfo/DWARF -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/include -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/include -fPIC -fPIC -fno-semantic-interposition -fvisibility-inlines-hidden -Werror=date-time -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wno-missing-field-initializers -pedantic -Wno-long-long -Wimplicit-fallthrough -Wno-maybe-uninitialized -Wno-class-memaccess -Wno-redundant-move -Wno-pessimizing-move -Wno-noexcept-type -Wdelete-non-virtual-dtor -Wsuggest-override -Wno-comment -Wmisleading-indentation -fdiagnostics-color -g -std=c++14  -fno-exceptions -fno-rtti -MD -MT lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFDebugRnglists.cpp.o -MF lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFDebugRnglists.cpp.o.d -o lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFDebugRnglists.cpp.o -c /mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/DebugInfo/DWARF/DWARFDebugRnglists.cpp
[1369/1678] /usr/bin/c++ -DGTEST_HAS_RTTI=0 -D_DEBUG -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/lib/LTO -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/LTO -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/include -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/include -fPIC -fPIC -fno-semantic-interposition -fvisibility-inlines-hidden -Werror=date-time -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wno-missing-field-initializers -pedantic -Wno-long-long -Wimplicit-fallthrough -Wno-maybe-uninitialized -Wno-class-memaccess -Wno-redundant-move -Wno-pessimizing-move -Wno-noexcept-type -Wdelete-non-virtual-dtor -Wsuggest-override -Wno-comment -Wmisleading-indentation -fdiagnostics-color -g -std=c++14  -fno-exceptions -fno-rtti -MD -MT lib/LTO/CMakeFiles/LLVMLTO.dir/LTOBackend.cpp.o -MF lib/LTO/CMakeFiles/LLVMLTO.dir/LTOBackend.cpp.o.d -o lib/LTO/CMakeFiles/LLVMLTO.dir/LTOBackend.cpp.o -c /mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/LTO/LTOBackend.cpp
[1370/1678] /usr/bin/c++ -DGTEST_HAS_RTTI=0 -D_DEBUG -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/lib/DebugInfo/DWARF -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/DebugInfo/DWARF -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/include -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/include -fPIC -fPIC -fno-semantic-interposition -fvisibility-inlines-hidden -Werror=date-time -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wno-missing-field-initializers -pedantic -Wno-long-long -Wimplicit-fallthrough -Wno-maybe-uninitialized -Wno-class-memaccess -Wno-redundant-move -Wno-pessimizing-move -Wno-noexcept-type -Wdelete-non-virtual-dtor -Wsuggest-override -Wno-comment -Wmisleading-indentation -fdiagnostics-color -g -std=c++14  -fno-exceptions -fno-rtti -MD -MT lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFTypeUnit.cpp.o -MF lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFTypeUnit.cpp.o.d -o lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFTypeUnit.cpp.o -c /mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/DebugInfo/DWARF/DWARFTypeUnit.cpp
[1371/1678] /usr/bin/c++ -DGTEST_HAS_RTTI=0 -D_DEBUG -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/lib/ObjectYAML -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/ObjectYAML -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/include -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/include -fPIC -fPIC -fno-semantic-interposition -fvisibility-inlines-hidden -Werror=date-time -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wno-missing-field-initializers -pedantic -Wno-long-long -Wimplicit-fallthrough -Wno-maybe-uninitialized -Wno-class-memaccess -Wno-redundant-move -Wno-pessimizing-move -Wno-noexcept-type -Wdelete-non-virtual-dtor -Wsuggest-override -Wno-comment -Wmisleading-indentation -fdiagnostics-color -g -std=c++14  -fno-exceptions -fno-rtti -MD -MT lib/ObjectYAML/CMakeFiles/LLVMObjectYAML.dir/yaml2obj.cpp.o -MF lib/ObjectYAML/CMakeFiles/LLVMObjectYAML.dir/yaml2obj.cpp.o.d -o lib/ObjectYAML/CMakeFiles/LLVMObjectYAML.dir/yaml2obj.cpp.o -c /mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/ObjectYAML/yaml2obj.cpp
[1372/1678] /usr/bin/c++ -DGTEST_HAS_RTTI=0 -D_DEBUG -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/lib/DebugInfo/DWARF -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/DebugInfo/DWARF -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/include -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/include -fPIC -fPIC -fno-semantic-interposition -fvisibility-inlines-hidden -Werror=date-time -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wno-missing-field-initializers -pedantic -Wno-long-long -Wimplicit-fallthrough -Wno-maybe-uninitialized -Wno-class-memaccess -Wno-redundant-move -Wno-pessimizing-move -Wno-noexcept-type -Wdelete-non-virtual-dtor -Wsuggest-override -Wno-comment -Wmisleading-indentation -fdiagnostics-color -g -std=c++14  -fno-exceptions -fno-rtti -MD -MT lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFAcceleratorTable.cpp.o -MF lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFAcceleratorTable.cpp.o.d -o lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFAcceleratorTable.cpp.o -c /mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/DebugInfo/DWARF/DWARFAcceleratorTable.cpp
[1373/1678] /usr/bin/c++ -DGTEST_HAS_RTTI=0 -D_DEBUG -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/lib/DebugInfo/DWARF -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/DebugInfo/DWARF -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/include -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/include -fPIC -fPIC -fno-semantic-interposition -fvisibility-inlines-hidden -Werror=date-time -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wno-missing-field-initializers -pedantic -Wno-long-long -Wimplicit-fallthrough -Wno-maybe-uninitialized -Wno-class-memaccess -Wno-redundant-move -Wno-pessimizing-move -Wno-noexcept-type -Wdelete-non-virtual-dtor -Wsuggest-override -Wno-comment -Wmisleading-indentation -fdiagnostics-color -g -std=c++14  -fno-exceptions -fno-rtti -MD -MT lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFDebugFrame.cpp.o -MF lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFDebugFrame.cpp.o.d -o lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFDebugFrame.cpp.o -c /mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/DebugInfo/DWARF/DWARFDebugFrame.cpp
[1374/1678] /usr/bin/c++ -DGTEST_HAS_RTTI=0 -D_DEBUG -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/lib/DebugInfo/DWARF -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/DebugInfo/DWARF -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/include -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/include -fPIC -fPIC -fno-semantic-interposition -fvisibility-inlines-hidden -Werror=date-time -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wno-missing-field-initializers -pedantic -Wno-long-long -Wimplicit-fallthrough -Wno-maybe-uninitialized -Wno-class-memaccess -Wno-redundant-move -Wno-pessimizing-move -Wno-noexcept-type -Wdelete-non-virtual-dtor -Wsuggest-override -Wno-comment -Wmisleading-indentation -fdiagnostics-color -g -std=c++14  -fno-exceptions -fno-rtti -MD -MT lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFExpression.cpp.o -MF lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFExpression.cpp.o.d -o lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFExpression.cpp.o -c /mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/DebugInfo/DWARF/DWARFExpression.cpp
[1375/1678] /usr/bin/c++ -DGTEST_HAS_RTTI=0 -D_DEBUG -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/lib/DebugInfo/DWARF -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/DebugInfo/DWARF -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/include -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/include -fPIC -fPIC -fno-semantic-interposition -fvisibility-inlines-hidden -Werror=date-time -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wno-missing-field-initializers -pedantic -Wno-long-long -Wimplicit-fallthrough -Wno-maybe-uninitialized -Wno-class-memaccess -Wno-redundant-move -Wno-pessimizing-move -Wno-noexcept-type -Wdelete-non-virtual-dtor -Wsuggest-override -Wno-comment -Wmisleading-indentation -fdiagnostics-color -g -std=c++14  -fno-exceptions -fno-rtti -MD -MT lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFFormValue.cpp.o -MF lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFFormValue.cpp.o.d -o lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFFormValue.cpp.o -c /mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/DebugInfo/DWARF/DWARFFormValue.cpp
[1376/1678] /usr/bin/c++ -DGTEST_HAS_RTTI=0 -D_DEBUG -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/lib/DebugInfo/DWARF -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/DebugInfo/DWARF -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/include -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/include -fPIC -fPIC -fno-semantic-interposition -fvisibility-inlines-hidden -Werror=date-time -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wno-missing-field-initializers -pedantic -Wno-long-long -Wimplicit-fallthrough -Wno-maybe-uninitialized -Wno-class-memaccess -Wno-redundant-move -Wno-pessimizing-move -Wno-noexcept-type -Wdelete-non-virtual-dtor -Wsuggest-override -Wno-comment -Wmisleading-indentation -fdiagnostics-color -g -std=c++14  -fno-exceptions -fno-rtti -MD -MT lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFListTable.cpp.o -MF lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFListTable.cpp.o.d -o lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFListTable.cpp.o -c /mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/DebugInfo/DWARF/DWARFListTable.cpp
[1377/1678] /usr/bin/c++ -DGTEST_HAS_RTTI=0 -D_DEBUG -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/lib/DebugInfo/GSYM -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/DebugInfo/GSYM -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/include -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/include -fPIC -fPIC -fno-semantic-interposition -fvisibility-inlines-hidden -Werror=date-time -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wno-missing-field-initializers -pedantic -Wno-long-long -Wimplicit-fallthrough -Wno-maybe-uninitialized -Wno-class-memaccess -Wno-redundant-move -Wno-pessimizing-move -Wno-noexcept-type -Wdelete-non-virtual-dtor -Wsuggest-override -Wno-comment -Wmisleading-indentation -fdiagnostics-color -g -std=c++14  -fno-exceptions -fno-rtti -MD -MT lib/DebugInfo/GSYM/CMakeFiles/LLVMDebugInfoGSYM.dir/LineTable.cpp.o -MF lib/DebugInfo/GSYM/CMakeFiles/LLVMDebugInfoGSYM.dir/LineTable.cpp.o.d -o lib/DebugInfo/GSYM/CMakeFiles/LLVMDebugInfoGSYM.dir/LineTable.cpp.o -c /mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/DebugInfo/GSYM/LineTable.cpp
[1378/1678] /usr/bin/c++ -DGTEST_HAS_RTTI=0 -D_DEBUG -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/lib/DebugInfo/GSYM -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/DebugInfo/GSYM -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/include -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/include -fPIC -fPIC -fno-semantic-interposition -fvisibility-inlines-hidden -Werror=date-time -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wno-missing-field-initializers -pedantic -Wno-long-long -Wimplicit-fallthrough -Wno-maybe-uninitialized -Wno-class-memaccess -Wno-redundant-move -Wno-pessimizing-move -Wno-noexcept-type -Wdelete-non-virtual-dtor -Wsuggest-override -Wno-comment -Wmisleading-indentation -fdiagnostics-color -g -std=c++14  -fno-exceptions -fno-rtti -MD -MT lib/DebugInfo/GSYM/CMakeFiles/LLVMDebugInfoGSYM.dir/FunctionInfo.cpp.o -MF lib/DebugInfo/GSYM/CMakeFiles/LLVMDebugInfoGSYM.dir/FunctionInfo.cpp.o.d -o lib/DebugInfo/GSYM/CMakeFiles/LLVMDebugInfoGSYM.dir/FunctionInfo.cpp.o -c /mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/DebugInfo/GSYM/FunctionInfo.cpp
[1379/1678] : && /mnt/vcpkg-ci/downloads/tools/cmake-3.27.1-linux/cmake-3.27.1-linux-x86_64/bin/cmake -E rm -f lib/libLLVMCore.a && /usr/bin/ar Dqc lib/libLLVMCore.a  lib/IR/CMakeFiles/LLVMCore.dir/AbstractCallSite.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/AsmWriter.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/Assumptions.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/Attributes.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/AutoUpgrade.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/BasicBlock.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/BuiltinGCs.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/Comdat.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/ConstantFold.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/ConstantRange.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/Constants.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/Core.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/DIBuilder.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/DataLayout.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/DebugInfo.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/DebugInfoMetadata.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/DebugLoc.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/DiagnosticHandler.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/DiagnosticInfo.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/DiagnosticPrinter.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/Dominators.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/FPEnv.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/Function.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/GCStrategy.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/GVMaterializer.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/Globals.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/IRBuilder.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/IRPrintingPasses.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/SSAContext.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/InlineAsm.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/Instruction.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/Instructions.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/IntrinsicInst.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/LLVMContext.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/LLVMContextImpl.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/LLVMRemarkStreamer.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/LegacyPassManager.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/MDBuilder.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/Mangler.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/Metadata.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/Module.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/ModuleSummaryIndex.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/Operator.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/OptBisect.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/Pass.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/PassInstrumentation.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/PassManager.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/PassRegistry.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/PassTimingInfo.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/PrintPasses.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/SafepointIRVerifier.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/ProfileSummary.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/PseudoProbe.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/ReplaceConstant.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/Statepoint.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/StructuralHash.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/Type.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/TypeFinder.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/Use.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/User.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/Value.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/ValueSymbolTable.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/VectorBuilder.cpp.o lib/IR/CMakeFiles/LLVMCore.dir/Verifier.cpp.o && /usr/bin/ranlib -D lib/libLLVMCore.a && :
[1380/1678] /usr/bin/c++ -DGTEST_HAS_RTTI=0 -D_DEBUG -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/lib/DebugInfo/GSYM -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/DebugInfo/GSYM -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/include -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/include -fPIC -fPIC -fno-semantic-interposition -fvisibility-inlines-hidden -Werror=date-time -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wno-missing-field-initializers -pedantic -Wno-long-long -Wimplicit-fallthrough -Wno-maybe-uninitialized -Wno-class-memaccess -Wno-redundant-move -Wno-pessimizing-move -Wno-noexcept-type -Wdelete-non-virtual-dtor -Wsuggest-override -Wno-comment -Wmisleading-indentation -fdiagnostics-color -g -std=c++14  -fno-exceptions -fno-rtti -MD -MT lib/DebugInfo/GSYM/CMakeFiles/LLVMDebugInfoGSYM.dir/InlineInfo.cpp.o -MF lib/DebugInfo/GSYM/CMakeFiles/LLVMDebugInfoGSYM.dir/InlineInfo.cpp.o.d -o lib/DebugInfo/GSYM/CMakeFiles/LLVMDebugInfoGSYM.dir/InlineInfo.cpp.o -c /mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/DebugInfo/GSYM/InlineInfo.cpp
[1381/1678] /usr/bin/c++ -DGTEST_HAS_RTTI=0 -D_DEBUG -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/lib/DebugInfo/DWARF -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/DebugInfo/DWARF -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/include -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/include -fPIC -fPIC -fno-semantic-interposition -fvisibility-inlines-hidden -Werror=date-time -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wno-missing-field-initializers -pedantic -Wno-long-long -Wimplicit-fallthrough -Wno-maybe-uninitialized -Wno-class-memaccess -Wno-redundant-move -Wno-pessimizing-move -Wno-noexcept-type -Wdelete-non-virtual-dtor -Wsuggest-override -Wno-comment -Wmisleading-indentation -fdiagnostics-color -g -std=c++14  -fno-exceptions -fno-rtti -MD -MT lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFDebugLine.cpp.o -MF lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFDebugLine.cpp.o.d -o lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFDebugLine.cpp.o -c /mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/DebugInfo/DWARF/DWARFDebugLine.cpp
[1382/1678] /usr/bin/c++ -DGTEST_HAS_RTTI=0 -D_DEBUG -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/lib/DebugInfo/DWARF -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/DebugInfo/DWARF -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/include -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/include -fPIC -fPIC -fno-semantic-interposition -fvisibility-inlines-hidden -Werror=date-time -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wno-missing-field-initializers -pedantic -Wno-long-long -Wimplicit-fallthrough -Wno-maybe-uninitialized -Wno-class-memaccess -Wno-redundant-move -Wno-pessimizing-move -Wno-noexcept-type -Wdelete-non-virtual-dtor -Wsuggest-override -Wno-comment -Wmisleading-indentation -fdiagnostics-color -g -std=c++14  -fno-exceptions -fno-rtti -MD -MT lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFDie.cpp.o -MF lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFDie.cpp.o.d -o lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFDie.cpp.o -c /mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/DebugInfo/DWARF/DWARFDie.cpp
[1383/1678] /usr/bin/c++ -DGTEST_HAS_RTTI=0 -D_DEBUG -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/lib/LTO -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/LTO -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/include -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/include -fPIC -fPIC -fno-semantic-interposition -fvisibility-inlines-hidden -Werror=date-time -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wno-missing-field-initializers -pedantic -Wno-long-long -Wimplicit-fallthrough -Wno-maybe-uninitialized -Wno-class-memaccess -Wno-redundant-move -Wno-pessimizing-move -Wno-noexcept-type -Wdelete-non-virtual-dtor -Wsuggest-override -Wno-comment -Wmisleading-indentation -fdiagnostics-color -g -std=c++14  -fno-exceptions -fno-rtti -MD -MT lib/LTO/CMakeFiles/LLVMLTO.dir/ThinLTOCodeGenerator.cpp.o -MF lib/LTO/CMakeFiles/LLVMLTO.dir/ThinLTOCodeGenerator.cpp.o.d -o lib/LTO/CMakeFiles/LLVMLTO.dir/ThinLTOCodeGenerator.cpp.o -c /mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/LTO/ThinLTOCodeGenerator.cpp
[1384/1678] /usr/bin/c++ -DGTEST_HAS_RTTI=0 -D_DEBUG -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/lib/LTO -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/LTO -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/include -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/include -fPIC -fPIC -fno-semantic-interposition -fvisibility-inlines-hidden -Werror=date-time -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wno-missing-field-initializers -pedantic -Wno-long-long -Wimplicit-fallthrough -Wno-maybe-uninitialized -Wno-class-memaccess -Wno-redundant-move -Wno-pessimizing-move -Wno-noexcept-type -Wdelete-non-virtual-dtor -Wsuggest-override -Wno-comment -Wmisleading-indentation -fdiagnostics-color -g -std=c++14  -fno-exceptions -fno-rtti -MD -MT lib/LTO/CMakeFiles/LLVMLTO.dir/LTO.cpp.o -MF lib/LTO/CMakeFiles/LLVMLTO.dir/LTO.cpp.o.d -o lib/LTO/CMakeFiles/LLVMLTO.dir/LTO.cpp.o -c /mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/LTO/LTO.cpp
[1385/1678] /usr/bin/c++ -DGTEST_HAS_RTTI=0 -D_DEBUG -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/lib/DebugInfo/GSYM -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/DebugInfo/GSYM -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/include -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/include -fPIC -fPIC -fno-semantic-interposition -fvisibility-inlines-hidden -Werror=date-time -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wno-missing-field-initializers -pedantic -Wno-long-long -Wimplicit-fallthrough -Wno-maybe-uninitialized -Wno-class-memaccess -Wno-redundant-move -Wno-pessimizing-move -Wno-noexcept-type -Wdelete-non-virtual-dtor -Wsuggest-override -Wno-comment -Wmisleading-indentation -fdiagnostics-color -g -std=c++14  -fno-exceptions -fno-rtti -MD -MT lib/DebugInfo/GSYM/CMakeFiles/LLVMDebugInfoGSYM.dir/GsymCreator.cpp.o -MF lib/DebugInfo/GSYM/CMakeFiles/LLVMDebugInfoGSYM.dir/GsymCreator.cpp.o.d -o lib/DebugInfo/GSYM/CMakeFiles/LLVMDebugInfoGSYM.dir/GsymCreator.cpp.o -c /mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/DebugInfo/GSYM/GsymCreator.cpp
[1386/1678] /usr/bin/c++ -DGTEST_HAS_RTTI=0 -D_DEBUG -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/lib/DebugInfo/GSYM -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/DebugInfo/GSYM -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/include -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/include -fPIC -fPIC -fno-semantic-interposition -fvisibility-inlines-hidden -Werror=date-time -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wno-missing-field-initializers -pedantic -Wno-long-long -Wimplicit-fallthrough -Wno-maybe-uninitialized -Wno-class-memaccess -Wno-redundant-move -Wno-pessimizing-move -Wno-noexcept-type -Wdelete-non-virtual-dtor -Wsuggest-override -Wno-comment -Wmisleading-indentation -fdiagnostics-color -g -std=c++14  -fno-exceptions -fno-rtti -MD -MT lib/DebugInfo/GSYM/CMakeFiles/LLVMDebugInfoGSYM.dir/GsymReader.cpp.o -MF lib/DebugInfo/GSYM/CMakeFiles/LLVMDebugInfoGSYM.dir/GsymReader.cpp.o.d -o lib/DebugInfo/GSYM/CMakeFiles/LLVMDebugInfoGSYM.dir/GsymReader.cpp.o -c /mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/DebugInfo/GSYM/GsymReader.cpp
[1387/1678] /usr/bin/c++ -DGTEST_HAS_RTTI=0 -D_DEBUG -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/lib/DebugInfo/DWARF -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/DebugInfo/DWARF -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/include -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/include -fPIC -fPIC -fno-semantic-interposition -fvisibility-inlines-hidden -Werror=date-time -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wno-missing-field-initializers -pedantic -Wno-long-long -Wimplicit-fallthrough -Wno-maybe-uninitialized -Wno-class-memaccess -Wno-redundant-move -Wno-pessimizing-move -Wno-noexcept-type -Wdelete-non-virtual-dtor -Wsuggest-override -Wno-comment -Wmisleading-indentation -fdiagnostics-color -g -std=c++14  -fno-exceptions -fno-rtti -MD -MT lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFUnit.cpp.o -MF lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFUnit.cpp.o.d -o lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFUnit.cpp.o -c /mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/DebugInfo/DWARF/DWARFUnit.cpp
[1388/1678] /usr/bin/c++ -DGTEST_HAS_RTTI=0 -D_DEBUG -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/lib/DebugInfo/GSYM -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/DebugInfo/GSYM -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/include -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/include -fPIC -fPIC -fno-semantic-interposition -fvisibility-inlines-hidden -Werror=date-time -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wno-missing-field-initializers -pedantic -Wno-long-long -Wimplicit-fallthrough -Wno-maybe-uninitialized -Wno-class-memaccess -Wno-redundant-move -Wno-pessimizing-move -Wno-noexcept-type -Wdelete-non-virtual-dtor -Wsuggest-override -Wno-comment -Wmisleading-indentation -fdiagnostics-color -g -std=c++14  -fno-exceptions -fno-rtti -MD -MT lib/DebugInfo/GSYM/CMakeFiles/LLVMDebugInfoGSYM.dir/DwarfTransformer.cpp.o -MF lib/DebugInfo/GSYM/CMakeFiles/LLVMDebugInfoGSYM.dir/DwarfTransformer.cpp.o.d -o lib/DebugInfo/GSYM/CMakeFiles/LLVMDebugInfoGSYM.dir/DwarfTransformer.cpp.o -c /mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/DebugInfo/GSYM/DwarfTransformer.cpp
[1389/1678] /usr/bin/c++ -DGTEST_HAS_RTTI=0 -D_DEBUG -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/lib/DebugInfo/DWARF -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/DebugInfo/DWARF -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/include -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/include -fPIC -fPIC -fno-semantic-interposition -fvisibility-inlines-hidden -Werror=date-time -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wno-missing-field-initializers -pedantic -Wno-long-long -Wimplicit-fallthrough -Wno-maybe-uninitialized -Wno-class-memaccess -Wno-redundant-move -Wno-pessimizing-move -Wno-noexcept-type -Wdelete-non-virtual-dtor -Wsuggest-override -Wno-comment -Wmisleading-indentation -fdiagnostics-color -g -std=c++14  -fno-exceptions -fno-rtti -MD -MT lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFContext.cpp.o -MF lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFContext.cpp.o.d -o lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFContext.cpp.o -c /mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/DebugInfo/DWARF/DWARFContext.cpp
[1390/1678] /usr/bin/c++ -DGTEST_HAS_RTTI=0 -D_DEBUG -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/lib/ObjectYAML -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/ObjectYAML -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/include -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/include -fPIC -fPIC -fno-semantic-interposition -fvisibility-inlines-hidden -Werror=date-time -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wno-missing-field-initializers -pedantic -Wno-long-long -Wimplicit-fallthrough -Wno-maybe-uninitialized -Wno-class-memaccess -Wno-redundant-move -Wno-pessimizing-move -Wno-noexcept-type -Wdelete-non-virtual-dtor -Wsuggest-override -Wno-comment -Wmisleading-indentation -fdiagnostics-color -g -std=c++14  -fno-exceptions -fno-rtti -MD -MT lib/ObjectYAML/CMakeFiles/LLVMObjectYAML.dir/ELFYAML.cpp.o -MF lib/ObjectYAML/CMakeFiles/LLVMObjectYAML.dir/ELFYAML.cpp.o.d -o lib/ObjectYAML/CMakeFiles/LLVMObjectYAML.dir/ELFYAML.cpp.o -c /mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/ObjectYAML/ELFYAML.cpp
[1391/1678] /usr/bin/c++ -DGTEST_HAS_RTTI=0 -D_DEBUG -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/lib/DebugInfo/DWARF -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/DebugInfo/DWARF -I/mnt/vcpkg-ci/buildtrees/llvm/x64-linux-dbg/include -I/mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/include -fPIC -fPIC -fno-semantic-interposition -fvisibility-inlines-hidden -Werror=date-time -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wno-missing-field-initializers -pedantic -Wno-long-long -Wimplicit-fallthrough -Wno-maybe-uninitialized -Wno-class-memaccess -Wno-redundant-move -Wno-pessimizing-move -Wno-noexcept-type -Wdelete-non-virtual-dtor -Wsuggest-override -Wno-comment -Wmisleading-indentation -fdiagnostics-color -g -std=c++14  -fno-exceptions -fno-rtti -MD -MT lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFVerifier.cpp.o -MF lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFVerifier.cpp.o.d -o lib/DebugInfo/DWARF/CMakeFiles/LLVMDebugInfoDWARF.dir/DWARFVerifier.cpp.o -c /mnt/vcpkg-ci/buildtrees/llvm/src/org-15.0.7-00acd5653b.clean/llvm/lib/DebugInfo/DWARF/DWARFVerifier.cpp
ninja: build stopped: subcommand failed.
```
</details>

**Additional context**

--- a/ports/llvm/0010-missing-include.patch
+++ b/ports/llvm/0010-missing-include.patch
@@ -1,0 +1,12 @@
+diff --git a/llvm/lib/Debuginfod/Debuginfod.cpp b/llvm/lib/Debuginfod/Debuginfod.cpp
+index ef4e11ca3..29fdd6ffb 100644
+--- a/llvm/lib/Debuginfod/Debuginfod.cpp
++++ b/llvm/lib/Debuginfod/Debuginfod.cpp
+@@ -40,6 +40,7 @@
+ #include "llvm/Support/xxhash.h"
+ 
+ #include <atomic>
++#include <thread>
+ 
+ namespace llvm {
+ static std::string uniqueKey(llvm::StringRef S) { return utostr(xxHash64(S)); }

--- a/ports/llvm/portfile.cmake
+++ b/ports/llvm/portfile.cmake
@@ -15,6 +15,7 @@ vcpkg_from_github(
         0007-Fix-install-bolt.patch
         0008-llvm_assert.patch
         0009-disable-libomp-aliases.patch
+        0010-missing-include.patch
 )
 
 vcpkg_check_features(

--- a/ports/llvm/vcpkg.json
+++ b/ports/llvm/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "llvm",
   "version": "15.0.7",
-  "port-version": 4,
+  "port-version": 5,
   "description": "The LLVM Compiler Infrastructure.",
   "homepage": "https://llvm.org",
   "license": "Apache-2.0",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -5026,7 +5026,7 @@
     },
     "llvm": {
       "baseline": "15.0.7",
-      "port-version": 4
+      "port-version": 5
     },
     "lmdb": {
       "baseline": "0.9.29",

--- a/versions/l-/llvm.json
+++ b/versions/l-/llvm.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "ce05511c7bf08ffecc627b73ae566e1a1e7e6b20",
+      "version": "15.0.7",
+      "port-version": 5
+    },
+    {
       "git-tree": "bd0b5a20abf736cc78645980e56569eae2e076ba",
       "version": "15.0.7",
       "port-version": 4

